subsys/mpsl/init/Kconfig

```diff
@@ -56,6 +56,14 @@ config MPSL_TRIGGER_IPC_TASK_ON_RTC_START_CHANNEL
 	  The application core part of the code needs to be aware of
 	  which channel is used.
 
+config MPSL_LOW_PRIO_IRQN
+	int
+	default $(dt_node_array_prop_int,$(dt_nodelabel_path,swi5),interrupts,0) if SOC_COMPATIBLE_NRF52X
```

Comment:
discussed OOB: I suggest to drop the dts complexity and "just hardcode" the number. 

When we do have to revisit this, I propose we do the right thing (IMO) and add a mpsl driver and configure the SWI through a property.